db/migrate/20170315095936_update_persistent_volumes_parent_type.rb

```diff
@@ -0,0 +1,7 @@
+class UpdatePersistentVolumesParentType < ActiveRecord::Migration[5.0]
+  def change
+    PersistentVolume.all.each do |pv|
```

Comment:
Unconditionally?  `PersistentVolume` ought to only return ones with type='PersistentVolume', and these ought to all belong to the EMS, but personally I'd feel safer adding a `.where(:parent_type => 'ManageIQ::Providers::ContainerManager')`.  If you don't, consider adding a migration spec verifying it doesn't touch container volumes.

`.all` could be inefficient / hit RAM limits, especially considering these were being multiplying...
`PersistentVolume.in_batches do` would be better, but here I believe you can just do them all in 1 DB query with 
http://apidock.com/rails/ActiveRecord/Relation/update_all